src/connectors/query-rules/connectQueryRules.ts

```diff
@@ -0,0 +1,237 @@
+import noop from 'lodash/noop';
+import isEqual from 'lodash/isEqual';
```

Comment:
do we not have babel-plugin-lodash here?

Reply:
We do not use it in the InstantSearch.js codebase.